docs/user/discover.asciidoc

```diff
@@ -285,6 +281,12 @@ To share your findings with a larger audience, click *Share* in the *Discover* t
 
 * <<discover-search-for-relevance, Search for relevance>>.
 
+* <<alerting-getting-started, Create an alert from Discover>>.
```

Comment:
Couple of thoughts on this bullet:
> * <<alerting-getting-started, Create an alert from Discover>>.
1) I think linking on this phrase is perhaps misleading, since that page won't tell you how to do the task in Discover. Can the link be at the end a la "Learn more"?
2) I think the preferred terminology is to "create a rule" not "create an alert", since the rule defines when/if alerts are generated.

Reply:
I'd suggest **alert rule** for that.

Reply:
Interestingly, we use "alert rule" in the ML app but seemingly nowhere else and not in the docs.  I'll try to suggest something in your latest text, @gchaps